packages/api-reference/src/types.ts

```diff
@@ -250,7 +250,19 @@ export type ReferenceLayoutSlot =
   | 'sidebar-start'
   | 'sidebar-end'
 
+export type ReferenceLayoutSlots = {
+  [x in ReferenceLayoutSlot]: (props: ReferenceSlotProps) => any
```

Comment:
do we need to specify any here? maybe a comment would help : )

Reply:
I'm not sure this is previous code that was moved.